--- package/feeds/turrispackages/omnia-support/Makefile.orig	2023-03-24 20:15:01.062985208 +0000
+++ package/feeds/turrispackages/omnia-support/Makefile	2023-03-24 18:30:11.115576908 +0000
@@ -58,7 +58,7 @@
 
 define Package/omnia-generic-support/install
 	$(INSTALL_DIR) $(1)/boot
-	$(INSTALL_DATA) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/dts/armada-385-turris-omnia-*.dtb "$(1)"/boot
+	$(INSTALL_DATA) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/dts/armada-385-turris-omnia*.dtb "$(1)"/boot
 	ln -s boot/boot.scr "$(1)"/boot.scr
 endef
 
